PyMPDATA/options.py

```diff
@@ -73,7 +73,10 @@ def dtype(self):
 
     @property
     def n_iters(self) -> int:
-        """number of iterations (1: upwind, 2: upwind + one corrective iteration, ...)"""
+        """Number of iterations in the MPDATA algorithm
```

Comment:
we could perhaps rename it to `n_corrective_iterations`?

Reply:
I've changed the phrasing a bit, should be better. I think we should stick to n_iters for now